cmd/remoterun/ssh_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2024 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remoterun
+
+import (
+	"bufio"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"math/big"
+	"net"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSSHForwarder(t *testing.T) {
```

Comment:
This test is kind of an integration one instead of a unit one, but it doesn't take too much time, and I think it is better to run it as a unit one, but let me know if you prefer to move it to an e2e one